crates/zkforge/bin/cmd/script/executor.rs

```diff
@@ -63,8 +63,12 @@ impl ScriptArgs {
         script_config.called_function = Some(func);
 
         // Only call the method if `setUp()` succeeded.
+        println!("result.success : {}", result.success);
         if result.success {
+            println!("address : {}", address);
+            println!("calldata : {:?}", calldata);
             let script_result = runner.script(address, calldata)?;
+            println!("script_result.success : {}", script_result.success);
```

Comment:
As you'll see if you run this code (following the PR description), setup (e.g. `result.success`) should be successful but the `script_result` will be false.